### DISCUSSION TITLE
Question about Date

### DISCUSSION THREAD

**Discussion by zu1k** [Planetable/Planet#327] [General]
I'm creating a small tool that works around `planet.json`, I found some Date type like `created` make me confused.

I think Date type should follow one of `RFC 2822` \ `RFC 3339` \ `ISO 8601`, this field should be readable, could you please make a change or give some kind of parsing specification ?

**Comment by livid** [comment]
Currently, it's directly from Swift JSON encoder for Swift's Date type, its raw form is a `TimeInterval` (=`Double`) of their milliseconds since the reference date: 12:00 a.m. UTC on January 1st, 2000.

**Comment by zu1k** [comment]
Thanks for your reply.

I am not familiar with swift, I wonder why Double, I think it should be some type of `int` if this means the count of milliseconds since a certain time.

Maybe it would be more appropriate to make changes in time for the early stages of the product.

```
decoder.dateDecodingStrategy = .iso8601
encoder.dateEncodingStrategy = .iso8601
```

**Comment by livid** [comment]
Thanks. But it would be a breaking change for previous versions.
